aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -30,3 +30,15 @@ proposals:
           transaction_shuffler_type:
             deprecated_sender_aware_v1: 32
           block_gas_limit : 35000
+  - name: enable_enable_storage_deletion_refund
+    metadata:
+      title: "Enable Storage Deletion Refund"
+      description: "AIP-32: Storage Deletion Refund"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/127"
+    execution_mode: SingleStep
```

Comment:
Multistep please

Reply:
So what does SingleStep mean? (The update_sequence has only one step?)

Reply:
No it uses a different cli and smart contract code path thats no longer preferred. Multistep works for when theres a single step

Reply:
Gotcha. If that's the case we should mark SingleStep deprecated?

Reply:
cc @sitalkedia  @lightmark  sorry